assets/src/domain/eventEditor/ui/datetimes/datesList/cardView/DateCard.tsx

```diff
@@ -6,23 +6,20 @@ import DateDetails from './DateDetails';
 import DateActionsMenu from '../actionsMenu/DateActionsMenu';
 
 import { DatetimeProvider } from '@edtrServices/context/DatetimeContext';
-import useDatetimeItem from '@edtrServices/apollo/queries/datetimes/useDatetimeItem';
 import statusBgColorClassName from '@sharedEntities/datetimes/helpers/statusBgColorClassName';
 
 import EntityCard from '@appLayout/EntityCard';
-import { EntityListItemProps } from '@appLayout/entityList';
+import type { EntityListItemProps } from '@appLayout/entityList';
 import { InlineEditHeading, InlineEditTextArea } from '@appInputs/InlineEditInput';
 import { useDatesListFilterState } from '@edtrServices/filterState';
 import { useDatetimeMutator } from '@edtrServices/apollo/mutations';
+import type { Datetime } from '@edtrServices/apollo/types';
```

Comment:
what's the benefit/reason for doing this?

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-exports